Ansible/ansible_collections/jfrog/platform/roles/postgres/tasks/install.yml

```diff
@@ -81,6 +81,16 @@
     timeout: 120
     sleep: 10
 
+- name: Assert database username and password are defined
```

Comment:
Could you please move this task to top of this file.? So that assertion will be done at the beginning of the playbook.